src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  display: block;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.logo__link {
+  width: 40px;
+  height: 40px;
+}
+
+.is-active::before {
+  content: "";
+  background-color: #00acdc;
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 4px;
+  border-radius: 2px;
+}
+
+.nav__link.is-active {
```

Comment:
Did you forget to fix it?
![image](https://user-images.githubusercontent.com/75496546/167779899-f829fb50-ba94-4f50-85dd-15267b309746.png)